Makefile

```diff
@@ -62,7 +66,9 @@ ensure_ns:
 run_local_tests: libdds-build
 	python3 -m unittest discover ${VERBOSE_TEST}
 
-curl_local:
+# We'd like to define a TEST_DEAL macro, but then the command-line output is
+# not as pretty.
+curl:
 	curl \
```

Comment:
Do we need three targets?

I would think two would be enough.

Reply:
I agree that two would be better! It will depend on what strategy we agree on for constructing the targets. If we do end up with three I'll make the local target the default.